.github/workflows/pr-merge.yml

```diff
@@ -0,0 +1,37 @@
+# This is a basic workflow to help you get started with Actions
+
+name: BZ Merge
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  pull_request_target:
+    branches:
+    - "*"
+    types:
+    - closed
+    
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in pnamearallel
+jobs:
+  # This workflow contains a single job called "build"
+  bz-on-pr-merge:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: docker://quay.io/konveyor/pr-merge-github-action
+        name: update bugzilla to modified
+        with:
+          bugzilla_token: ${{ secrets.BUGZILLA_TOKEN }}
+          org_repo: ${{ github.repository }}
+          pr_number: ${{ github.event.pull_request.number }}
+          bz_product: "Migration Toolkit for Containers"
+          title: ${{ github.event.pull_request.title }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          branch_to_release: "release-1.3.2:1.3.z,release-1.4.0:1.4.0,master:1.5.0"
```

Comment:
Should master be for 1.4.z bugs?

Reply:
@shawn-hurley , I wondered that too, but ultimately went with the same mapping for `master:1.5.0` as you used for mig-controller [pr-merge.yml#L36](https://github.com/konveyor/mig-controller/blob/master/.github/workflows/pr-merge.yml#L36).  If guidance is to use 1.4.z, I'll update where applicable to reflect that on the PRs across repos.

Reply:
I think it would be good to ask @eriknelson or @ibolton336 how they are using the master branch and bugs for this particular repo